src/DomainServer.ts

```diff
@@ -15,11 +15,16 @@
  *
  *  @namespace DomainServer
  *  @property {boolean} isConnected - <code>true</code> if connected to the domain, <code>false</code> if not connected.
+ *      <em>Read-only.</em>
  */
-const DomainServer = new (class {
+const DomainServer = new class {
 
-    isConnected = false;
+    #_isConnected = false;
```

Comment:
We want to use `#`s so that private members aren't exposed in the public API.
And we want to use `protected` for some internal classes (so I went with using `private` for internal classes).
However, I'm currently thinking of changing all `private`s to `#`s.

Reply:
Reading the docs (https://www.typescriptlang.org/docs/handbook/2/classes.html#private and https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Classes/Private_class_fields#private_instance_fields) I realize that "private" is TypeScript compile time checking and "#" is runtime. JSDoc is supposed to know about both. Visually, I prefer using "private" (and possibly JSDoc's "@ignore") but, if there is a need for the ES6 private fields then it is just a question of consistency.

Reply:
I think that it's a good idea to use `#`s for public APIs to reduce that likelihood of misuse (unintentional and intentional) of the API. While `private` is nice visually, I've found that `#` is also nice while working in the code because it provides extra immediate information on the scope of methods.
I'm currently intending to switch back to using `#`s for private fields and methods ... probably after my current work on WebRTC connections to assignment clients.